pull_flutter/lib/views/settings.dart

```diff
@@ -43,6 +39,20 @@ class _PullSettingsPageState extends ConsumerState<SettingsPage> {
                 launchUrl(Uri.parse('https://pulldating.tips'));
               },
             ),
+            ListTile(
+              title: Text('Licenses',
+                  style: GoogleFonts.nunito(
```

Comment:
I'd prefer we didn't have to define the text style for each individual item. Can you add a parent TextTheme parameter to the `nunitoTextTheme` in`main.dart`, maybe make this an h5 or something, so that we can just use`Theme.of(context).textTheme.headline5`?

Reply:
Added a TextTheme parameter for h5 headlines